### DISCUSSION TITLE
Display schema of response

### DISCUSSION THREAD

**Discussion by gtolarc** [scalar/scalar#645] [Ideas]
It would be nice to see the response schema displayed. I don't know if this is a bug, but currently only the request shows the schema.

**Comment by marcklingen** [comment]
+1, scalar seems great but as our API returns nested objects, an example is not enough for users to understand what to expect

**Comment by marclave** [comment]
I bring good news @gtolarc and @marcklingen , this has been done in the latest scalar release `@scalar/api-reference@1.13.3`

✨
